engine/Makefile

```diff
@@ -22,11 +22,41 @@ test_spark_intro:
 
 tests: test_ada_intro test_spark_intro
 
+publish-staging:
+	@echo "Publishing current branch to learn-staging..."
+	@if [ ! -d learn-staging-html-pages ] ;\
+	then \
+		git clone -b gh-pages git@github.com:AdaCore/learn-staging-html-pages.git; \
+		cd learn-staging-html-pages; \
+	else \
+		cd learn-staging-html-pages; \
+		git checkout gh-pages; \
+		git pull; \
+	fi \
+
+	# remove all files from the repo to get rid of stale copies
+	@rm -rf *
+
+	# reset files that are needed that won't be copied from the build
+	@git reset CNAME
+
+	# copy build files to learn-staging-html-pages
+	@cp -R $(BUILDDIR)/html/. .
+
+	# commit and push new changes
```

Comment:
Is that on purpose ? (commented lines ?)

Reply:
I seem to have commited a testing version of the Makefile rather than the final version. Will fix now.